### PR TITLE
Polysemy: trochlea of femur. Obsolete offending UBERON:7500077

### PR DESCRIPTION
**Pull request by vasvir**
This PR tries to resolve the 3 of 4 of the issues mentioned in #2424

The problem that is addressed here is that

"trochlea of femur" (UBERON:7500077) is an exact synonym to "intercondylar fossa" (UBERON:4200096). This creates unecessary polysemy.

The proposed solution is to obsolete UBERON:7500077 altogether.

I also moved from the obsoleted term the following:
* synonym: "patellar surface of femur" EXACT []
* is_a: UBERON:4200230 ! surface of bone
* relationship: part_of UBERON:0004406 ! distal epiphysis of femur

Tagged @chandron, @elekka since he helped with the biology.

Tagged @matentzn as requested.

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -222349,12 +222352,11 @@ property_value: dcterms-date 2021-05-28T19:11:21Z xsd:dateTime
...
 id: UBERON:7500077
-name: trochlea of femur
+name: obsolete trochlea of femur
 def: "A trochlea on the distal end of a femur that interacts with the patella." [https://www.imaios.com/en/e-Anatomy/Anatomical-Parts/patellar-surface]
-synonym: "patellar surface of femur" EXACT []
-is_a: UBERON:4200230 ! surface of bone
-relationship: part_of UBERON:0004406 ! distal epiphysis of femur
 property_value: dcterms-date 2021-06-02T00:19:07Z xsd:dateTime
+is_obsolete: true
+replaced_by: UBERON:UBERON:4200096
```
Comment (matentzn) [inline src/ontology/uberon-edit.obo:222359]: Just `UBERON:4200096`

**Review by matentzn — CHANGES_REQUESTED** [review]

**Comment by vasvir** [comment]
Thanks for the heads up. I believe I fixed it now (after a brief git 101 detour).

**Comment by matentzn** [comment]
Cool, many of our curators are on vacation, they will deal with this issue here soon! Thanks :)

**Review by nicolevasilevsky — APPROVED** [review]
this looks good to me

**Comment by nicolevasilevsky** [comment]
I don't understand why this is failing? @shawntanzk could you help or point me to someone that can? Thanks!

**Comment by vasvir** [comment]
I would also like to be enlightened here. It fails in "post_comment / Post reasoned post_comment" and at least one of my previous PR had the same issue. So it would be great to understand:

Is this a problem of the a) PR or with the b) CI?

If a) how to fix?

Thanks

**Comment by shawntanzk** [comment]
@nicolevasilevsky & @vasvir looks like a github token issue I think - might be something we overlooked @ubyndr, if someone doesn't have the right permissions and they create the PR, it might be an issue. 
Btws don't worry too much about it, that is just to create and post a human readable diff to the ticket, as long as ontology_qc passes, its safe :)

**Comment by nicolevasilevsky** [comment]
@shawntanzk is this okay to merge then? thanks!

**Comment by shawntanzk** [comment]
> @shawntanzk is this okay to merge then? thanks!

Didn't look at content, but QC wise its good to merge :)

**Comment by nicolevasilevsky** [comment]
I will merge!

**Pull request merged**